test/Main.purs

```diff
@@ -5,19 +5,21 @@ import Prelude
 import Data.Maybe (Maybe(..))
```

Comment:
Need to add more test to cover all the new functions but and as already mentioned would have been cool to use quickcheck, just need to work out Erlang Floats 😄